### DISCUSSION TITLE
Node-RED MCU Edition mentioned in-print in Japan!

### DISCUSSION THREAD

**Discussion by phoddie** [phoddie/node-red-mcu#42] [Announcements]
Check out the introduction of Node-RED MCU Edition in I/O Magazine (I think!) in Japan shared by @kitazaki [on Twitter](https://twitter.com/Zakkiea/status/1593040981448077313)!

![Fhud1o8aUAEufN4](https://user-images.githubusercontent.com/6856458/202520138-2ac12bd0-43a8-43ca-a62b-cb38bf29ff1e.jpeg)

**Comment by kitazaki** [comment]
@phoddie 
Thank you for sharing the topic in I/O magazine.
I will also share the booklet on this weekend (20, Nov 2022) at 7th Gisho Expo.
https://gishohaku.dev/
![Node-RED-MCU入門_v1_1](https://user-images.githubusercontent.com/23477407/202593395-155417a9-0153-48f6-88d8-6e1d2dfec67f.png)

**Reply by phoddie** [reply]
The third part of the series has been [published](https://twitter.com/Zakkiea/status/1615145522825945088). Very nice. Congratulations to @kitazaki.

![FmomSNKaAAU4-ay](https://user-images.githubusercontent.com/6856458/212963010-76f2c120-4fcb-4e63-9828-060de39af77b.jpeg)
